machine-runner/src/runner/runner.ts

```diff
@@ -164,19 +159,11 @@ export type PersistFn<RegisteredEventsFactoriesTuple extends Readonly<MachineEve
 export const createMachineRunner = <
   SwarmProtocolName extends string,
   MachineName extends string,
-  RegisteredEventsFactoriesTuple extends Readonly<MachineEvent.Factory.Any[]>,
   Payload,
 >(
   sdk: Actyx,
-  tags: Tags<MachineEvent.Factory.ReduceToEvent<RegisteredEventsFactoriesTuple>>,
-  initialFactory: StateFactory<
-    SwarmProtocolName,
-    MachineName,
-    RegisteredEventsFactoriesTuple,
-    any,
-    Payload,
-    any
-  >,
+  tags: Tags<MachineEvent.Any>,
```

Comment:
doesn’t this mean that now any `Tags` would work that has a `type: string` property?

Reply:
Yes it does